Cargo.toml

```diff
@@ -13,7 +13,7 @@ readme = "README.md"
 winrt-macros = { path = "crates/macros",  version = "0.8.0" }
 winrt-gen = { path = "crates/gen",  version = "0.8.0" }
 const-sha1 = "0.2"
-winmd = "0.6.0"
+winrt-winmd = { path = "crates/winmd",  version = "0.8.0" }
```

Comment:
Why the change of name?

Reply:
I ran into some trouble with the name when I was porting but you're right the simpler name should suffice - switching back to `winmd`.